recipes/pyubxutils/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "pyubxutils" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/pyubxutils-{{ version }}.tar.gz
+  sha256: 52932b67773ee104029aed1be8806082ef2dae1a28262415fb731bc3fc5a8538
+build:
+  entry_points:
+    - ubxsetrate = pyubxutils.ubxsetrate:main
+    - ubxsave = pyubxutils.ubxsave:main
+    - ubxload = pyubxutils.ubxload:main
+    - ubxsimulator = pyubxutils.ubxsimulator_cli:main
+    - ubxcompare = pyubxutils.ubxcompare:main
+
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - setuptools >=66.0.0
+    - wheel
+    - pip
+  run:
+    - python {{ python_min }}
```

Comment:
```suggestion
    - python >={{ python_min }}
```

Reply:
just a suggestion, but maybe change the linter message to say explicitly:

`you should usually use python >={{ python_min }} for the python entry`

Reply:
It should be `python {{ python_min }}` in host, to ensure you are building with the minimum supported python, and `python >={{ python_min }}` in run so one can install this in any supported python. The linter message is correct. Please remove the `>=` from host.